### PR TITLE
Artwork highlight back-end (+ Tailwind admin components)

### PR DESCRIPTION
**Pull request by eronisko**
# Description
image
image

This change adds a new back-end section ("Featured artworks") written almost entirely using Tailwind.

Also adds a few new components:
* Vue: Autocomplete -- renderless component with Bloodhound
* Vue: QueryString -- simple utility component for updating URL
* Blade components for admin forms: form, button, label, input, alert.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

# Checklist:

- [x] I have updated the [CHANGELOG](../CHANGELOG.md)
- [x] I have requested at least 1 reviewer for this PR
- [ ] I have made corresponding changes to the documentation

### CODE REVIEW

File: resources/views/dielo.blade.php [hunk 1]
```diff
@@ -51,13 +51,7 @@
...
                     <h1 class="nadpis-dielo" itemprop="name">{{ $item->title }}</h1>
                     <h2 class="inline">
-                        @php
-                            $authors = $item->authorities
-                                ->toBase()
-                                ->merge($item->getAuthorsWithoutAuthority());
-                        @endphp
-
-                        @foreach($authors as $author)
+                        @foreach($item->authors_with_authorities as $author)
```
Comment (igor-kamil) [inline resources/views/dielo.blade.php:54]: 👌

File: tests/Models/ItemTest.php [hunk 2]
```diff
@@ -148,6 +148,27 @@ public function testMergedAuthorityNamesAndAuthors()
         $this->assertEquals('Philips Wouwerman', $data['author'][1]);
     }
 
+    public function testAuthorsWithAuthoritiesAttribute()
```
Comment (igor-kamil) [inline tests/Models/ItemTest.php:151]: 👏

**Review by igor-kamil — APPROVED** [review]
wow! respect for your effort. nice and massive PR 😄

File: app/FeaturedArtwork.php [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+    public function item()
+    {
+        return $this->belongsTo(Item::class);
+    }
+
+    public function getIsPublishedAttribute() {
+        return !!$this->published_at;
+    }
+
+    public function setIsPublishedAttribute(bool $isPublished) {
```
[the remark this answers is not in the document]
Comment (eronisko) [reply app/FeaturedArtwork.php:27]: Unfortunately, we don't have `Publishable` trait here (yet). I've considered this but we'd have to consolidate the various models first. Some have a boolean `publish`, some `published_at` and some `published_date`.

File: resources/js/components/Autocomplete.vue [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+        :options="options"
+        :filterable="false"
+        :label="optionLabel"
+        :clearable="false"
+        :value="value"
+        :placeholder="placeholder"
+        v-on:input="(value) => $emit('input', value)"
+    >
+        <template v-slot:option="option">
+            <slot name="option" v-bind="option"> {{ option.id }}aaa </slot>
```
[the remark this answers is not in the document]
Comment (eronisko) [reply resources/js/components/Autocomplete.vue:13]: Good catch :)

File: resources/views/components/admin/a.blade.php [hunk 5]
```diff
@@ -0,0 +1 @@
+<a {!! $attributes->merge(['class' => 'tw-text-[#428bca] hover:tw-underline']) !!}>{{ $slot }}</a>
```
[the remark this answers is not in the document]
Comment (eronisko) [reply resources/views/components/admin/a.blade.php]: Good point, renamed

**Pull request merged**